client/components/theme-collection/index.tsx

```diff
@@ -83,6 +88,9 @@ export default function ThemeCollection( {
 					<div className="theme-collection__description">{ description }</div>
 				</div>
 				<div className="theme-collection__carousel-controls">
+					<Button className="theme-collection__see-all" onClick={ onSeeAll }>
```

Comment:
I wonder why a regular link won't work. 🤔

Reply:
A plain link does work, but we're going to want to track these clicks. When I work on the tracking issue this implementation might change depending on what I find during the implementation.